ksg_nett/settings.py

```diff
@@ -163,3 +163,9 @@
 
 STATIC_ROOT = 'static/'
 MEDIA_ROOT = 'media/'
+
+# GLOBAL SETTINGS
+# ------------------------------
+MINIMUM_SOCI_AMOUNT = 0
+SOCI_CARD_ID = 8008135  # For identifying Soci's own Soci Bank Account (a normal card id is 9 digits)
+DIRECT_CHARGE_SKU = "X-BELØP"
```

Comment:
The file needs an encoding declaration if we want the 'Ø' there, i.e. `coding: utf-8`

Reply:
Thanks for the tip. We could also just not use Ø for this constant. What do you prefer?

Reply:
`X-BELOP` might perhaps be more portable? I don't have any strong opinions otherwise.